src/commands/deploy.rs

```diff
@@ -200,11 +200,15 @@ impl DeployCommand {
 
     async fn compute_buildinfo(&self, cfg: &RawAppManifest) -> Result<BuildMetadata> {
         let mut sha256 = Sha256::new();
+        let mut app_folder = PathBuf::from(&self.app);
+        app_folder.pop();
 
         for x in cfg.components.iter() {
             match &x.source {
                 config::RawModuleSource::FileReference(p) => {
-                    let mut r = File::open(p)?;
+                    let mut absolute_path = PathBuf::from(&app_folder);
```

Comment:
Nit: this isn't necessarily an absolute path.  `full_path` maybe?

Reply:
Updated it. Thank you.